src/main/kotlin/blackJack/view/InputView.kt

```diff
@@ -1,22 +1,45 @@
 package blackJack.view
 
 import blackJack.domain.Player
+import blackJack.domain.Players
 
 object InputView {
-    fun inputPlayer(): List<String> {
-        println("게임에 참여할 사람의 이름을 입력하세요. (쉼표 기준으로 분리)")
-        return checkNullOrBlank(readLine()).split(",").map { it.trim() }
+    fun playerNames(): Players {
+        println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)")
+        val inputValue = readLine()!!
+        val names = checkBlank(inputValue)
+        return Players(names.split(",").map { it.trim() })
     }
 
-    fun inputWhether(player: Player): String {
-        println("${player.name}님은 한장의 카드를 더 받겠습니까? (예는 y, 아니오는 n)")
-        return checkNullOrBlank(readLine())
+    fun bettingMoney(player: Player): Int {
+        println("${player.name}의 배팅 금액은?")
+        return changeInt(readLine()!!)
     }
 
-    private fun checkNullOrBlank(names: String?): String {
-        if (!names.isNullOrBlank()) {
-            return names
+    fun hitOrStay(player: Player): String {
+        println("${player.name}는 한장의 카드를 더 받겠습니까?(예는 y, 아니오는 n)")
+        return checkString(readLine()!!.trim())
+    }
+
+    private fun checkBlank(inputValue: String): String {
+        if (inputValue.isNotBlank()) {
+            return inputValue
+        }
+        throw IllegalArgumentException("공백 값을 입력하지 마세요")
+    }
+
+    private fun changeInt(inputValue: String): Int {
+        try {
+            return inputValue.toInt()
+        } catch (e: Exception) {
+            throw IllegalArgumentException("$inputValue 는 숫자가 아닙니다.")
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return inputValue.toIntOrNull() ?: throw IllegalArgumentException("$inputValue 는 숫자가 아닙니다.")
```